### PR TITLE
Fix resnet parameter copy error

### PR DESCRIPTION
**Pull request by cyoon1729**
#255

### CODE REVIEW

File: rl_algorithms/common/abstract/distributed_worker.py [hunk 1]
```diff
@@ -108,7 +114,7 @@ def compute_priorities(self, experience: Dict[str, np.ndarray]):
         pass
 
     @abstractmethod
-    def synchronize(self, new_params: list):
+    def synchronize(self, new_params_list: List[str], new_params: List[np.ndarray]):
```
Comment (MrSyee) [inline rl_algorithms/common/abstract/distributed_worker.py:117]: should fix wrong argument

File: rl_algorithms/common/abstract/distributed_worker.py [hunk 2]
```diff
@@ -32,15 +33,20 @@ def step(self, action: np.ndarray) -> Tuple[np.ndarray, np.float64, bool, dict]:
         pass
 
     @abstractmethod
-    def synchronize(self, new_params: list):
+    def synchronize(self, param_name_list: List[str], new_params: list):
```
Comment (MrSyee) [inline rl_algorithms/common/abstract/distributed_worker.py:36]: should fix wrong argument

File: rl_algorithms/common/abstract/distributed_worker.py (hunk 1 above)
Comment (cyoon1729) [reply rl_algorithms/common/abstract/distributed_worker.py]: Fixed in new commit!

File: rl_algorithms/common/abstract/distributed_worker.py (hunk 2 above)
Comment (cyoon1729) [reply rl_algorithms/common/abstract/distributed_worker.py]: Fixed in new commit!

**Pull request merged**